tools/pypi_rss/main.go

```diff
@@ -0,0 +1,224 @@
+package main
+
+import (
+	"context"
+	"encoding/xml"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"regexp"
+	"slices"
+	"time"
+
+	"github.com/google/oss-rebuild/internal/taskqueue"
+	"github.com/google/oss-rebuild/internal/urlx"
+	"github.com/google/oss-rebuild/pkg/feed"
+	"github.com/google/oss-rebuild/pkg/rebuild/rebuild"
+	"github.com/google/oss-rebuild/pkg/rebuild/schema"
+	"github.com/google/oss-rebuild/tools/benchmark"
+	"github.com/pkg/errors"
+)
+
+var (
+	versionURLRegex = regexp.MustCompile(`https://pypi\.org/project/([^/]+)/([^/]+)/`)
+	apiURI          = flag.String("api", "", "OSS Rebuild API endpoint URI")
+	executionMode   = flag.String("execution-mode", "attest", "[attest|smoketest] the mode in which to execute rebuilds for incoming python updates")
+	taskQueuePath   = flag.String("task-queue", "", "the path identifier of the task queue to use")
+	taskQueueEmail  = flag.String("task-queue-email", "", "the email address of the serivce account Cloud Tasks should authorize as")
+	refBench        = flag.String("benchmark", "", "reference benchmark describing the packages we're interested in")
```

Comment:
hmm not sure what "reference" means here. can we remove it? "benchmark defining the set of tracked packages" might be how i'd write it